schema-loader/README.md

```diff
@@ -356,15 +356,15 @@ By default, the schema loader enables auto-scaling of RU for all tables: RU is s
 
 When creating tables for a JDBC database with this tool, Scalar DB data types are converted to RDB-specific data types as shown below.
 
-| ScalarDB | MySQL | PostgreSQL | Oracle | SQL Server |
-| ---- | ---- |  ---- |  ---- |  ---- | 
-| INT | INT | INT | INT | INT |
-| BIGINT | BIGINT | BIGINT | NUMBER(19) | BIGINT |
+| ScalarDB | MySQL | PostgreSQL | Oracle         | SQL Server |
+| ---- | ---- |  ---- |----------------|  ---- | 
+| INT | INT | INT | INT            | INT |
+| BIGINT | BIGINT | BIGINT | NUMBER(19)     | BIGINT |
 | TEXT | LONGTEXT | TEXT | VARCHAR2(4000) | VARCHAR(8000) |
-| FLOAT | DOUBLE | FLOAT | BINARY_FLOAT | FLOAT(24) |
-| DOUBLE | DOUBLE | DOUBLE PRECISION | BINARY_DOUBLE | FLOAT |
-| BOOLEAN | BOOLEAN | BOOLEAN | NUMBER(1) | BIT |
-| BLOB | LONGBLOB | BYTEA | BLOB | VARBINARY(8000) |
+| FLOAT | DOUBLE | FLOAT | BINARY_FLOAT   | FLOAT(24) |
+| DOUBLE | DOUBLE | DOUBLE PRECISION | BINARY_DOUBLE  | FLOAT |
+| BOOLEAN | BOOLEAN | BOOLEAN | NUMBER(1)      | BIT |
+| BLOB | LONGBLOB | BYTEA | RAW(2000)      | VARBINARY(8000) |
```

Comment:
Since Oracle does not natively support the column type alteration from `BLOB` (Scalar DB BLOB regular column type) to `RAW(64)` (Scalar DB BLOB primary key/index column type) which is something we wanted to be able to do if a column were to converted to a secondary index. 
We decided to change the regular default column type from `BLOB` to `RAW(2000)`.